cloudinit/config/schema.py

```diff
@@ -435,11 +454,11 @@ def get_parser(parser=None):
 
 def handle_schema_args(name, args):
     """Handle provided schema args and perform the appropriate actions."""
-    exclusive_args = [args.config_file, args.docs]
+    exclusive_args = [args.config_file, args.docs, args.system]
     if not any(exclusive_args) or all(exclusive_args):
```

Comment:
I think this check needs updating: `all(exclusive_args)` will be `False` if any two of the three are passed:

```py
In [3]: exclusive_args = ["config_file", None, "system"]                                                                                                                                                                                                                                  

In [4]: if not any(exclusive_args) or all(exclusive_args): 
   ...:     print('Expected one of --config-file, --system or --docs arguments') 
   ...:      
In [5]:
```

Reply:
Good find. Changed logic and added unit tests